src/DevChatter.DevStreams.Web/Pages/Index.cshtml

```diff
@@ -27,11 +30,52 @@
             <li>BrandonSatrom</li>
         </ul>
     </div>
+    <div class="col-md-12 text-center">
+        <div id="twitch-embed"></div>
+    </div>
 </div>
 
-@section Scripts
-{
+@section Scripts{
     <link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/fullcalendar/3.9.0/fullcalendar.min.css' />
     <script src='https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.22.2/moment.min.js'></script>
     <script src='https://cdnjs.cloudflare.com/ajax/libs/fullcalendar/3.9.0/fullcalendar.min.js'></script>
+    <script src="https://embed.twitch.tv/embed/v1.js"></script>
+
+    <script type="text/javascript">
```

Comment:
How about if we extract this code into a JavaScript file for all homepage interactivity. I'd recommend putting it in `/wwwroot/js/vue/` in a file like 'home.js` or 'index.js` or `home-page.js` or something like that.